mypy.ini

```diff
@@ -1,3 +1,4 @@
 [mypy]
 exclude = (vendor*)/$
 follow_imports = skip
+mypy_path = flare_splunk_integration/bin/vendor
```

Comment:
Why is this here? The mypy we're expecting to use should be in `venv-tools/bin`, shouldn't it?

Reply:
this configuration is to let it know where to find dependencies if they're not able to find them with the normal paths; that's how I fixed my issue of Mypy not finding the flareio dependency in the flare_external_requests.py imports

Reply:
Interesting.